java/kotlin/BUILD.bazel

```diff
@@ -50,10 +50,23 @@ kt_jvm_library(
     deps = ["//java/core"],
 )
 
+java_import(
+    name = "kotlin-stdlib-jvm",
```

Comment:
Why is this indirection necessary?

Reply:
It's not necessary, and I could remove it; the target changed within `@com_github_jetbrains_kotlin`, and I didn't want to expose the `java_import` etc functionality. I felt it was cleaner to have a single alias used in both `//java/kotlin` and `//java/kotlin-lite`.

But this is a matter of taste and I would defer to Google and/or other advice :)

Reply:
Unless you mean the `java_import`? That is necessary because the target is now a JAR file, there is no `kotlin-stdlib` in that dependency anymore. Rules Kotlin was upgraded because Rules JVM was upgraded, which happened because the earlier version depends on toolchain internals which changed with Bazel 7.x.

Reply:
Sorry, @haberman; maybe you didn't get these because I forgot to tag. My mentions is where I'm usually looking.